scvi/external/solo/_model.py

```diff
@@ -257,20 +345,29 @@ def auto_forward(module, x):
         for _, tensors in enumerate(scdl):
             x = tensors[_CONSTANTS.X_KEY]
             pred = auto_forward(self.module, x)
-            if not soft:
```

Comment:
@adamgayoso why did you remove this?

Reply:
Finally getting back to work on things. So hopefully I can get that tutorial out soon